### PR TITLE
Moved source files from webots repository

### PR DESCRIPTION
**Pull request by omichel**
👍

### CODE REVIEW

**Comment by stefaniapedrazzi** [comment]
`src/doc/dia/.gitignore` (ignoring generated *.pdf and *pdf.png) is not copied

**Comment by omichel** [comment]
I added the `.ignore` files.
I will remove all these files in https://github.com/omichel/webots/pull/4478

👍

**Comment by stefaniapedrazzi** [comment]
I'm not sure about the folder renaming, i.e. using the program name instead of the format.
The `svg` files can be modified using other tools than `Inkscape`.

Then, the `formula` folder name was more understandable than `tex`, given that the TEX file names doesn't specify that they contain formulas.

**Comment by stefaniapedrazzi** [comment]
By the now `tex`is the only folder that uses a format name instead of a program name.

**Comment by omichel** [comment]
`tex` is a program name. Although in this case, it is probably one of its friends that is used, e.g. `pdftex` or `latex`.

**Comment by omichel** [comment]
I renamed the folders following your remarks.

👍

File: src/dia/Makefile [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+ tutorial_4_wheels_lowlevel.pdf  \
+ tutorial_4_wheels_lowlevel.pdf.png  \
+ tutorial_ackermann_vehicle.pdf \
+ tutorial_ackermann_vehicle.pdf.png \
+ controller_plugin.pdf \
+ controller_plugin.pdf.png
+
+install : all
+	cp $(REF_TARGETS) ../../../doc/reference/pdf
+	cp $(GUIDE_TARGETS) ../../../doc/guide/pdf
```
Comment (stefaniapedrazzi) [inline src/dia/Makefile]: These pathe seem to refer to the old documentation

File: src/Makefile [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	@echo "  make math                  for the math formulas"
+	@echo "  make clean                 remove useless files"
+
+dia: dia/*.dia
+	@echo "Producing diagrams..."
+	@make install -C dia
+
+math: math/*.tex
+	@echo "Producing math formulas..."
+	@make install -C math
```
Comment (stefaniapedrazzi) [inline src/Makefile]: I get an error when executing `make math` in `src` folder.
But it works fine if I execute `make` directly in the `math` folder.

**Comment by omichel** [comment]
That should be all fixed.

👍

**Pull request merged**